### PR TITLE
Add workflow-guidelines

### PR DESCRIPTION
**Pull request by ireneontheway**
Add workflow-guidelines

### CODE REVIEW

**Comment by ireneontheway** [comment]
@Anonymitaet Hello,can you help review this file and merge it ?

**Comment by Anonymitaet** [comment]
Please replace "crowdin" with "Crowdin".

File: workflow_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+   
+- Green: translated and approved  
+- Blue: translated but not approved 
+- Grey: not translated
+
+![](media/translation-status.jpg)
+
+Based on your knowledge about Pulsar, you can translate the content that you are familiar with first.
+
+After choosing the file to translate, please check it out in [Translation schedule](https://github.com/apache/pulsar-translation/blob/master/schedule.md). If you find someone is translating this file, please choose another one. If not, you can create a PR and add the file  name to [Translation schedule](https://github.com/apache/pulsar-translation/blob/master/schedule.md). Then you can submit the PR.
```
Comment (Anonymitaet) [inline workflow_guidelines.md]:
```suggestion
After choosing the file to translate, please check it out in [Translation schedule](https://github.com/apache/pulsar-translation/blob/master/schedule.md). If you find someone is translating this file, please choose another one. If not, you can create a PR and add the file name to [Translation schedule](https://github.com/apache/pulsar-translation/blob/master/schedule.md). Then you can submit the PR.
```

File: workflow_guidelines.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+### Label Setting
+- Reviewers : No need to set up. Reviewers will receive the notification of PR you submitted automatically and will designate someone to handle it.
```
Comment (Anonymitaet) [inline workflow_guidelines.md]:
```suggestion
- Reviewers: No need to set up. Reviewers will receive the notification of PR you submitted automatically and will designate someone to handle it.
```

File: workflow_guidelines.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+- Assignees : Choose yourself to receive notifications.
+- Label : Choose “translation”.
```
Comment (Anonymitaet) [inline workflow_guidelines.md]:
```suggestion
- Label: Choose “translation”.
```

File: workflow_guidelines.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+![](media/translation-status.jpg)
+
+Based on your knowledge about Pulsar, you can translate the content that you are familiar with first.
+
+After choosing the file to translate, please check it out in [Translation schedule](https://github.com/apache/pulsar-translation/blob/master/schedule.md). If you find someone is translating this file, please choose another one. If not, you can create a PR and add the file  name to [Translation schedule](https://github.com/apache/pulsar-translation/blob/master/schedule.md). Then you can submit the PR.
+
+### Label Setting
+- Reviewers : No need to set up. Reviewers will receive the notification of PR you submitted automatically and will designate someone to handle it.
+- Assignees : Choose yourself to receive notifications.
```
Comment (Anonymitaet) [inline workflow_guidelines.md]:
```suggestion
- Assignees: Choose yourself to receive notifications.
```

File: workflow_guidelines.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+
+### Review
+After finishing a translation, please leave a comment in your PR.  : 
+For example, I've finished the translation, could you help review?
+
+After receiving the comment, reviewers start to review. The reviewer can make simple comments and notify the translator. Direct discussion in PR is encouraged.
+
+If there are no problems, the reviewer will approve this PR and merge it.
+
+The following image gives a general workflow. The workflow source file is available  at [lucidchat](https://www.lucidchart.com/invitations/accept/0ebad9d8-ddf3-4a92-8ee6-e813a9bc58ff).
```
Comment (Anonymitaet) [inline workflow_guidelines.md]:
```suggestion
The following image gives a general workflow. The workflow source file is available at [lucidchat](https://www.lucidchart.com/invitations/accept/0ebad9d8-ddf3-4a92-8ee6-e813a9bc58ff).   
```

File: workflow_guidelines.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+
+> ####Note
+> You don’t need to add spaces behind Chinese punctuations, even if they are followed by English words.
+
+###  3. Chinese punctuation
+
+
+Please use Chinese punctuations in Chinese documents: You should use “、”  rather than  “，” when you translate English into Chinese. For example, “Kubernetes, Meso, Docker“ should be translated to “Kubernetes、Mesos、Docker“. Vice versa.
+
+Dealing with different items : Please use (。) at the end of one sentence and use (；) at the end of one item. If each item involves one sentence, please use (。) at the end of each item.
```
Comment (Anonymitaet) [inline workflow_guidelines.md]:
```suggestion
Dealing with different items: Please use (。) at the end of one sentence and use (；) at the end of one item. If each item involves one sentence, please use (。) at the end of each item.
```

**Comment by ireneontheway** [comment]
I've updated. If it is OK,please help merge it.Thanks

**Comment by ireneontheway** [comment]
I press the Assignee button by mistake. If it's necessary, can you assign this pr to me again?@Anonymitaet

**Comment by Anonymitaet** [comment]
@ireneontheway some of my comments have not been incorporated, any thoughts?

**Comment by ireneontheway** [comment]
I‘ve incorporated them. @Anonymitaet

**Pull request merged**